definitions/procedures/packages/update_all_confirmation.rb

```diff
@@ -17,10 +17,10 @@ def run
           version and/or operating system using '#{command} update'.
           To upgrade to the next #{feature(:instance).product_name} version use '#{command} upgrade'.
           Do you want to proceed with update of everything regardless of
-          the recommendations?
+          the recommendations
```

Comment:
While I get why you're dropping the `?` here, every other occasion of `ask_decision` uses a message that ends with a question-mark, and then adds the question.
Conceptually, I'd then expect the possibly answers on a new line, but that seems not to happen right now.

Reply:
> Conceptually, I'd then expect the possibly answers on a new line, but that seems not to happen right now.

Can you maybe provide example of what you think the output should look like?

Reply:
```
This action will potentially eat a small kitten.
You should be using foreman-mrawntain to avoid this!
Do you want to proceed with update of everything regardless of the recommendations?
[y(yes), n(no), q(let me see if I have enough spare kittens)]
```

Reply:
You can (and I wouldn't be opposed to that), but I also didn't really want to stop you from doing a release ;)

Maybe, keep the `?`, assert the output is `the recommendations?, [y(yes), q(quit)]` (thus matching the other places we use decisions), refactor in a follow up?

(But again: this has an ack, and it's me being pedantic. feel free to merge as is and do more cleanups later)